Source/WebCore/platform/ios/VideoPresentationInterfaceIOS.h

```diff
@@ -126,27 +125,27 @@ class VideoPresentationInterfaceIOS
     };
 
     RefPtr<VideoPresentationModel> videoPresentationModel() const { return m_videoPresentationModel.get(); }
-    virtual bool shouldExitFullscreenWithReason(ExitFullScreenReason) = 0;
+    virtual bool shouldExitFullscreenWithReason(ExitFullScreenReason);
     HTMLMediaElementEnums::VideoFullscreenMode mode() const { return m_currentMode.mode(); }
     WEBCORE_EXPORT virtual bool mayAutomaticallyShowVideoPictureInPicture() const = 0;
     void prepareForPictureInPictureStop(Function<void(bool)>&& callback);
     WEBCORE_EXPORT void applicationDidBecomeActive();
     bool inPictureInPicture() const { return m_enteringPictureInPicture || m_currentMode.hasPictureInPicture(); }
     bool returningToStandby() const { return m_returningToStandby; }
 
-    virtual void willStartPictureInPicture() = 0;
-    virtual void didStartPictureInPicture() = 0;
-    virtual void failedToStartPictureInPicture() = 0;
+    virtual void willStartPictureInPicture();
+    virtual void didStartPictureInPicture();
+    virtual void failedToStartPictureInPicture();
     void willStopPictureInPicture();
-    virtual void didStopPictureInPicture() = 0;
-    virtual void prepareForPictureInPictureStopWithCompletionHandler(void (^)(BOOL)) = 0;
+    virtual void didStopPictureInPicture();
+    virtual void prepareForPictureInPictureStopWithCompletionHandler(void (^)(BOOL));
     virtual bool isPlayingVideoInEnhancedFullscreen() const = 0;
 
     WEBCORE_EXPORT void setMode(HTMLMediaElementEnums::VideoFullscreenMode, bool shouldNotifyModel);
     void clearMode(HTMLMediaElementEnums::VideoFullscreenMode, bool shouldNotifyModel);
     bool hasMode(HTMLMediaElementEnums::VideoFullscreenMode mode) const { return m_currentMode.hasMode(mode); }
-#if PLATFORM(WATCHOS) || PLATFORM(APPLETV)
-    UIViewController *presentingViewController();
+#if PLATFORM(WATCHOS) || PLATFORM(APPLETV) || PLATFORM(VISION)
+    WEBCORE_EXPORT UIViewController *presentingViewController();
```

Comment:
I know why this is here, but I don't think this change is necessary. Nothing in `PLATFORM(VISION)` calls `presentingViewController()` anymore.